DuckDuckGo/BrokenSiteInfo.swift

```diff
@@ -66,7 +67,8 @@ public struct BrokenSiteInfo {
                           Keys.os: UIDevice.current.systemVersion,
                           Keys.manufacturer: "Apple",
                           Keys.model: UIDevice.current.model,
-                          Keys.gpc: AppDependencyProvider.shared.appSettings.sendDoNotSell ? "true" : "false"]
+                          Keys.gpc: AppDependencyProvider.shared.appSettings.sendDoNotSell ? "true" : "false",
+                          Keys.ampUrl: LinkCleaner.shared.getLastAmpUrl() ?? ""]
```

Comment:
We could pass AMP state as a init parameter to avoid having that as a singleton.